packages/rds/initial-bootstrap.ts

```diff
@@ -0,0 +1,90 @@
+import { getDatabaseCredentials } from "./auth/secrets";
+import { createConnection, Connection } from "typeorm";
+
+import { CloudFormationCustomResourceEvent, CloudFormationCustomResourceResponse } from "aws-lambda";
+
+type User = {
+  name: string;
+  dbPrivileges: string[];
+};
+
+const USERS: User[] = [
+  {
+    name: "atat-api-read",
+    dbPrivileges: ["CONNECT", "SELECT"],
+  },
+  {
+    name: "atat-api-write",
+    dbPrivileges: ["CONNECT", "SELECT", "INSERT", "UPDATE", "DELETE"],
+  },
+  {
+    name: "atat-api-admin",
+    dbPrivileges: ["ALL PRIVILEGES"],
+  },
+];
+
+async function connectWithoutDatabase(): Promise<Connection> {
+  // This always gets set by the infrastructure code.
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const dbAuth = await getDatabaseCredentials(process.env.DATABASE_SECRET_NAME!);
+  return await createConnection({
+    type: "postgres",
+    host: process.env.DATABASE_HOST,
+    port: 3306,
```

Comment:
3306 is for mysql, right?
```suggestion
    port: 5432,
```